charts/psmdb-operator/README.md

```diff
@@ -19,15 +19,15 @@ To install the chart with the `psmdb` release name using a dedicated namespace (
 
 ```sh
 helm repo add percona https://percona.github.io/percona-helm-charts/
-helm install my-operator percona/psmdb-operator --version 1.13.0 --namespace my-namespace
+helm install my-operator percona/psmdb-operator --version 1.14.0 --namespace my-namespace
```

Comment:
Could we fix k8s versions too? 
`Kubernetes 1.19+`

Reply:
Thanks, fixed!